operator/config/manager/kustomization.yaml

```
@@ -10,5 +10,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: docker.io/odubajdt/keptn-lifecycle-operator
-  newTag: latest
+  newName: keptn-lifecycle-operator
+  newTag: 0.1.1-amd64
```

you'll need the `v` here. I checked, and the 0.1.0 images are also prefixed with `v`
```suggestion
  newTag: v0.1.1-amd64
```

---

these are overridden every time we invoke `make`. In the build, they will get the right value